department-of-veterans-affairs/caseflow
# Use form generator for RAMP election form page

Resolves #16318 

### Description
Use form generator for RAMP election form in intake

### Acceptance Criteria
- [ ] Validation still works for receipt date and opt in election form

### Testing Plan
1. Go to `/intake`
2. Select 'RAMP Opt in Election Form'
3. Search for veteran that has RAMP elections e.g. 582074255S,has_ramp_election_with_contentions
4. Verify validation works for receipt date and opt in election radio buttons 

### User Facing Changes
 N/A
